include/json_struct/json_struct.h

```diff
@@ -6464,7 +6464,15 @@ inline parse_string_error parseNumber(const char *number, size_t size, parsed_st
     }
     else
     {
-      if (NoDigitCount || parsedString.significand_digit_count < 19)
+      bool localNoDigitCount = false;
+
+      // Must separate this to make a constexpr if-statement and avoid warnings:
+      JS_IF_CONSTEXPR(NoDigitCount)
+      {
+        localNoDigitCount = true;
+      }
+
+      if (localNoDigitCount || parsedString.significand_digit_count < 19)
```

Comment:
right, but then we loose the constexpr bit. Maybe we can just change the template parameter to a function argument?